db/migrate/20230223075620_add_column_to_posts.rb

```diff
@@ -0,0 +1,8 @@
+class AddColumnToPosts < ActiveRecord::Migration[7.0]
+  def change
+    add_column :posts, :title, :string
+    add_column :posts, :text, :string
```

Comment:
- [ ] Same here the `text` should be a `text`, not a `string`, kindly make sure to change the text type to `text` so your migrations reflect the `ERD`